Pilot/MessageSender.py

```diff
@@ -15,6 +15,8 @@
 
 import Queue
 import logging
+import requests
```

Comment:
those imports have been hidden, cause not all the nodes have stomp and requests installed.
In principle only RESTMessage sender requires requests, and STOMP Message sender requires stomp.
I don't know if we want to add it explicitely?

Reply:
OK I re-added the late import of them.